pandas/plotting/_matplotlib/hist.py

```diff
@@ -60,7 +60,7 @@ def __init__(
         self.bins = bins  # use mpl default
         self.bottom = bottom
         # Do not call LinePlot.__init__ which may fill nan
-        MPLPlot.__init__(self, data, **kwargs)
+        MPLPlot.__init__(self, data, **kwargs)  # pylint: disable=non-parent-init-called
```

Comment:
this comment is useful - for the other two cases, can we move the comment to be directly above the `MPLPlot.__init__` line (or add it there if it's not already present, as in the Kdeplot case?)

Reply:
Thanks, @MarcoGorelli . I added comments as you suggested.